e2e-tests/tests/reserve/conftest.py

```diff
@@ -0,0 +1,188 @@
+from pytest import fixture
+from config.api_config import ApiConfig
+from src.blockchain_api import BlockchainApi
+from src.cardano_cli import cbor_to_bech32, hex_to_bech32
+from src.partner_chains_node.models import Reserve, VFunction
+import json
+import logging
+
+
+@fixture(scope="session")
+def governance_address(config: ApiConfig) -> str:
+    return config.nodes_config.governance_authority.mainchain_address
+
+
+@fixture(scope="session")
+def payment_key(config: ApiConfig, governance_skey_with_cli):
+    return config.nodes_config.governance_authority.mainchain_key
+
+
+@fixture(scope="session")
+def governance_vkey_bech32(config: ApiConfig):
+    vkey = config.nodes_config.governance_authority.mainchain_pub_key
+    vkey_bech32 = hex_to_bech32(vkey, "addr_vk")
```

Comment:
Addresses I've observed on testnets start with `addr_test1`

Reply:
see my comment above, `addr_vk` is bech32 HRP. This is how you do it
`echo <vkey> | bech32 addr_vk`